drivers/sensors/nau7802.c

```diff
@@ -124,20 +125,20 @@ static int nau7802_read_reg(FAR nau7802_dev_s *dev, uint8_t addr, void *buf,
                             uint8_t nbytes)
 {
   struct i2c_msg_s readcmd[2] = {
```

Comment:
Please move "{" to new line. The nxstyle in CI should have catch it early, it seems faulty, I will fix it.